news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.8.3"
+subtitle: "Maintenance version of the 1.7.8.x branch"
+date:   2022-01-26 14:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2022/01/1.7.8.3_banner.jpg
+icon: icon-checked
+tags:
+- version
+- patch
+- releases
+- 1.7.8.x
+- 1.7
+
+---
+
+PrestaShop 1.7.8.3 is now available. This maintenance release fixes 25 bugs and 1 security issue since the 1.7.8.2 release.
+Note: two of these fixes are not fully backward compatible ([#26933](https://github.com/PrestaShop/PrestaShop/issues/26933) and [#26643](https://github.com/PrestaShop/PrestaShop/issues/26643)) 
```

Comment:
```suggestion
Note: two of these fixes are not fully backward compatible:
- [#24987](https://github.com/PrestaShop/PrestaShop/issues/24987): 
  -  LegacyModuleExtractor now requires an extra parameter catalogueExtractExcludedDirectories to define folders to exclude 
  - In LegacyModuleExtractor we use specific class for phpExtractor, smartyExtractor and twigExtractor instead of ExtractorInterface, because the excludedDirs feature is in the TraitExtractor and not defined in the Interface
- [#26643](https://github.com/PrestaShop/PrestaShop/issues/26643) : 
  - If the website theme is based on classic theme (template and style), then the menu and searchbar style can be broken
```